packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -38,8 +40,12 @@ class HttpResponseError {
   /// Used by the platform implementation to create a new [HttpResponseError].
   const HttpResponseError({
     required this.statusCode,
+    this.request
   });
 
   /// The HTTP status code.
   final int statusCode;
+
+  /// The associated request.
+  final WebResourceRequest? request;
```

Comment:
I think you could replace this with the `LoadRequestParams` class that already exists.